checkout_sdk/exception.py

```diff
@@ -7,6 +7,8 @@
 class CheckoutException(Exception):
 
     def __init__(self, message=None):
```

Comment:
I don't know Python that well, but can't you just have the default set to empty string here rather than using the conditional below?

Reply:
None is the Python equivalent of “no value” or “undefined” (similar to null in other languages). Using None allows the SDK to handle optional parameters, deferred initialisation, and missing data gracefully without introducing ambiguous default values.
One important reason for using None in exception classes is to avoid undesired behavior when converting an empty message to a string. If None is passed as the message and directly converted to a string (e.g., str(None)), Python will produce the string "None", which is likely unintended and confusing for users.